### PR TITLE
docs: hide banner

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese | -          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Review by yoyo837 — APPROVED** [review]

**Comment by afc163** [comment]
冲突了

**Comment by li-jia-nan** [comment]
> 冲突了

update

File: .dumi/theme/layouts/GlobalLayout.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useCallback, useEffect, useMemo } from 'react';
+import React, { useCallback, useEffect, useMemo, Suspense } from 'react';
+import dayjs from 'dayjs';
```
Comment (afc163) [inline .dumi/theme/layouts/GlobalLayout.tsx]: 'dayjs' is defined but never used. (@typescript-eslint/no-unused-vars)

Comment (li-jia-nan) [reply .dumi/theme/layouts/GlobalLayout.tsx]: 唉？这个我记得我已经改了

**Review by afc163 — APPROVED** [review]

**Pull request merged**